modules/mod_feed/tmpl/default.php

```diff
@@ -127,7 +127,7 @@
 						<?php
 							// Strip the images.
 							$text = OutputFilter::stripImages($text);
-							$text = HTMLHelper::_('string.truncate', $text, $params->get('word_count', 0));
+							$text = HTMLHelper::_('string.truncate', $text, $params->get('char_count', 0));
```

Comment:
```suggestion
							$text = HTMLHelper::_('string.truncate', $text, $params->get('char_count', $params->get('word_count', 0) * 5));
```

I would suggest to use the average letter count for english words and multiply it with the old parameter, as default for the new parameter.

Reply:
That is not a good idea as it is tied to english and it would confuse a user who inputs 5 words and sees 3 words

Reply:
It's better then a string with `0` characters for upgraded sites.

In this context word count is not optimal anyway. But if you don't like it you can easily fix it by explode the string using whitespace and only take the word count.

Reply:
If you are concerned about the upgraded sites then I can always revert the field name change as already stated. FYI the 0 means all not none

Reply:
I suspect that we already have enough "silent" b/c breaks of this kind without automatically migration path.